packages/markdown/components/CodeTabs/index.jsx

```diff
@@ -1,6 +1,8 @@
 const React = require('react');
 const PropTypes = require('prop-types');
 
+const langRemap = require('./lang-remap.json');
```

Comment:
Instead of this, you can use `@readme/syntax-highlighter/uppercase` and do:

```
{meta || uppercase(lang)}
```

Reply:
Ahhh, that’s great. Will update accordingly.

Reply:
Maybe that uppercase method should be split out into separate library 🤔